.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Lint Python code'
+
+on: [push, pull_request]
+
+env:
+  PYTHON_VERSION: 3.6
```

Comment:
Maybe we should use matrix strategy & test against 3.6 (cur) / 3.8 (next?) / 3.10 (future)

Reply:
Using matrix now